validators/TextStringValidator.js

```diff
@@ -0,0 +1,9 @@
+const _ = require("lodash")
+
+function languageValidator(language) {
+    return _.includes(["hebrew", "english"], language)
```

Comment:
Regarding this - see previous comments - this should be defined in one places only and then sent to the client etc

Reply:
Here in the BE you can reference it directly though, or call a function which returns this array, doesn't matter